mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala

```diff
@@ -131,7 +131,7 @@ class RowMatrix(
       throw new IllegalArgumentException(s"Argument with more than 65535 cols: $cols")
     }
     if (cols > 10000) {
-      val mem = cols * cols * 8
+      val mem = (math.pow(cols, 2) * 8).formatted("%,.0f")
```

Comment:
Surely I can do it.  I was worrying some one may ask for the accurate number in the PR~~ :)

Another thing is, /125000 might should be changed to >> 15?